### ISSUE TITLE
About useful research with [exa.ai](https://exa.ai/)

### ISSUE THREAD

**Issue by fdrskh** [fdrskh/Reasysearch#2]
Don’t you hate it when you cannot find what you want while googling?
It’s over for you now.

Searching for information with Exa is advantageous over using a regular browser due to the following reasons:

1. Semantic Understanding: Exa uses a model which interprets the meaning of the users queries, unlike conventional keyword-based search engines such as the Google. This does so in a fashion that enables Exa to present results tailored to the actual content and the meaning of the user inquiry and consequently produce accurate and relevant information.

2. Neural Search: Exa is a neural engine that processes search queries using the human language that behaves intelligently without compromising the level of the complexity in queries. While performing filtering, it omits the noisy and irrelevant websites, thus, people end up enjoying high-quality and useful information.

3. Customizable Embeddings: Users can get webpage content swiftly just by reviewing their selected embedding models which are called "highlights." This feature allows users to have easy search results because they can easily get specific information.

4. Information Filtering: Among others, the merit of Exa's screening algorithm lies in the fact that it not only uses a search by meaning but also by the meaning of the whole query, leading to correct and detailed knowledge that the user requires. With this go, the duller operating served by the search engines is removed since obtuse search results that are found in traditional searchers are no more.

5. Regular Updates: Through the efforts of Exa, the index is updated every 2 minutes via high-quality web page resulting in the users having access to a new and improved dynamic environment where they are able to identify the best available information on the internet.

Summing up, Exa has semantic understanding, smart search, customizable embeddings, filtering out data, and regular updates that are better features for searching for information than usual browsers or search engines, such as Google.

And, there is one more insanely powerful tool here, and it’s the search for exact types of sources, like you see right here. PDFs, GitHubs, Tweets, Movies, it’s a great additon to be able to narrow down the range of your research, this is really useful, so be sure to check it out.
